feature/mypage/src/main/java/com/teamoffroad/feature/mypage/presentation/component/DiaryTimeDialog.kt

```diff
@@ -0,0 +1,137 @@
+package com.teamoffroad.feature.mypage.presentation.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Dialog
+import androidx.compose.ui.window.DialogProperties
+import com.teamoffroad.core.designsystem.component.clickableWithoutRipple
+import com.teamoffroad.core.designsystem.theme.Main2
+import com.teamoffroad.core.designsystem.theme.Main3
+import com.teamoffroad.core.designsystem.theme.OffroadTheme
+import com.teamoffroad.core.designsystem.theme.White
+
+@Composable
+fun DiaryTimeDialog(
+    title: String,
+    content: String,
+    cancelButtonText: String,
+    nextButtonText: String,
+    modifier: Modifier = Modifier,
+    shape: Shape = RoundedCornerShape(14.dp),
+    onClick: () -> Unit,
+    onClickCancel: () -> Unit,
+    isNext: Boolean = true
+) {
+    Dialog(
+        onDismissRequest = { onClickCancel() },
```

Comment:
```suggestion
        onDismissRequest = { onCancelClick() },
```
c: 위 네이밍이 좀 더 적절할 것 같아요!

Reply:
수정했습니다